src/main/java/org/spongepowered/common/event/tracking/phase/GeneralPhase.java

```diff
@@ -147,7 +149,13 @@ public GeneralPhase addChild(TrackingPhase child) {
     public void unwind(CauseTracker causeTracker, IPhaseState state, PhaseContext phaseContext) {
         if (state == State.COMMAND) {
             final ICommandSender sender = phaseContext.getSource(ICommandSender.class)
```

Comment:
This was added to bleeding just recently, so I'll just drop that part of the PR now.